common/src/main/java/io/pravega/common/concurrent/ExecutorServiceHelpers.java

```diff
@@ -99,6 +99,9 @@ public static Snapshot getSnapshot(ExecutorService service) {
         } else if (service instanceof ForkJoinPool) {
             val fjp = (ForkJoinPool) service;
             return new Snapshot(fjp.getQueuedSubmissionCount(), fjp.getActiveThreadCount(), fjp.getPoolSize());
+        } else if (service instanceof ThreadPoolScheduledExecutorService) {
```

Comment:
Codecov reports this isn't tested. You need to add some unit tests in this package (for this class) to cover these lines of code.